python/pipeline/experiment.py

```diff
@@ -593,4 +593,35 @@ class AutoProcessing(dj.Manual):
     autosegment=false   :boolean       # segment somas in the first channel with default method
     """
 
+@schema
+class Projector(dj.Lookup):
+    definition = """
+    # projector specifications
+    projector_id        : tinyint                               # projector id
+    refresh_rate        : tinyint                               # refresh rate in Hz
+    ---
+    red                 : enum('UV', 'Green', 'Blue', 'None')   # color to be used for red channel
```

Comment:
I generally don't like enums because they are hard to extend. Consider adding a lookup table with colors and use that here
```
-> Color.proj(red_color='color') # color to be used for red channel
```

Reply:
I think using enum is fine... if we need to extend later, one can easily go back to SQL and add a color if needed...

Reply:
can you? at some point I was told changing enums will be annoying. Even if it's doable, I doubt it is recommended relying on altering the table structure when new changes come along, using enum is reasonable when there is no reasonable thought that it will be extended later.